deps.edn

```diff
@@ -24,4 +24,4 @@
            :deploy {:extra-deps {deps-deploy/deps-deploy {:mvn/version "0.0.9"}}
                     :main-opts ["-m" "deps-deploy.deps-deploy" "deploy" "replikativ-konserve.jar"]}
            :format {:extra-deps {cljfmt/cljfmt {:mvn/version "0.7.0"}}
-                    :main-opts ["-m" "cljfmt.main" "check"]}}}
\ No newline at end of file
+                    :main-opts ["-m" "cljfmt.main" "fix"]}}}
```

Comment:
That's not good for the CI that is relying on it...You would have to duplicate the entry here to have it as an alias.

Reply:
ah ok, CI is not checking this in konserve, so probably fine here.

Reply:
Good catch, that was not intended to be committed.